### ISSUE TITLE
Selector for #id can be read as a color

### ISSUE THREAD

**Issue by ionnsachie** [xkeshav/color-collector#10]
Hey,

Great extension, fairly minor quibble on id selectors but if you have one that starts with a valid color i.e. `#abfoo` it will try to extract this as a variable to the root as well. 

Example:
```CSS
body{
    color: #fff;
}

#feature{
    display: block;
}

#abba{
    display: block;
}
```
```CSS
/* following :root block added by css color collector */
:root {
	--body__txt--1: #fff;
	--body__txt--2: #fea;
	--feature__defaultElement--3: #abba;
}

body{
    color: var(--body__txt--1);
}

var(--body__txt--2)ture{
    display: block;
}

var(--feature__defaultElement--3){
    display: block;
}
```

VS-Code Version: 1.78.0 
css-color-collector: v1.8.0

Thanks

**Comment by xkeshav** [comment]
Thanks for the feedback. I see the issue and  will make changes to fix.

**Comment by xkeshav** [comment]
Hi @ionnsachie 
issue has been fixed in v 1.9.0 and now live. 
although there is known isue when there are multiple id selector which resemble the hex color code, then it will convert all selector except last one but in your case, it has been fixed both of issue. kindly check
